README.md

```diff
@@ -1,2 +1,52 @@
 # fprime-openmct
 Scripts to connect the F Prime ground system to OpenMCT
+
+## Installation Instructions.
+OpenMCT runs a server using Node JS, so Node must be installed. Node Version 18.17.0 works and is stable with OpenMCT.
+### Install NVM (Node Version Manager)
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+```
+### Install Node 
+```
+nvm install v18.17.0
+nvm alias default v18.17.0 # Make v18.17.0 default if you have multiple Node Versions on your system
+```
+### Install F-Prime OpenMCT Dependencies and OpenMCT from Source
+```
+npm install # from the root of the repository
```

Comment:
this should be ran from the src/fprime_openmct/javascript directory since the package.json exists there (rather than at the root)